app/com/kinja/play/plugins/Closure.scala

```diff
@@ -45,7 +43,7 @@ class InvalidClosureValueException(obj: Any, path: Option[String] = None) extend
 /**
  * Play plugin for Closure.
  */
-class ClosurePlugin(app: Application) extends Plugin {
+class ClosurePlugin @Inject() (implicit app: Application) extends Plugin {
```

Comment:
On the side note: Plugins are deprecated in Play 2.4: https://www.playframework.com/documentation/2.4.x/api/scala/index.html#play.api.Plugin

```
(Since version 2.4.0) Use modules instead. See play.api.inject.Module
```